src/netcdf_scm/iris_cube_wrappers.py

```diff
@@ -457,6 +474,8 @@ def load_data_from_identifiers(self, **kwargs):
             filepath of the file to load and the variable constraint to use.
         """
         with warnings.catch_warnings(record=True) as w:
+            # iris v2.2.0 under py3.7 raises a DeprecationWarning
```

Comment:
do you know what for? i.e. is just ignoring it a good idea?

Reply:
They use the `collections` module throughout their codebase rather than rather than `collections.abc`. It just a DeprecationWarning, but it messes with the tests since we are interested in capturing warnings.